GirafIntegrationTest/tests/test_authorization.py

```diff
@@ -633,9 +681,12 @@ def test_auth_PUT_weektemplate_id_should_fail(self):
 
         Endpoint: PUT:/v1/WeekTemplate/{id}
         """
-        response = put(f'{BASE_URL}v1/WeekTemplate').json()
-        self.assertFalse(response['success'])
-        self.assertEqual(response['errorKey'], 'NotFound')
+        response = put(f'{BASE_URL}v1/WeekTemplate/123')
+        response_body = response.json()
+
+        
```

Comment:
One newline too much here. PyCharm has a neat, PEP-compliant linter that catches minor things like this.